project/Dependencies.scala

```diff
@@ -10,7 +10,7 @@ object Dependencies {
   val scala211 = "2.11.12"
   val scala212 = "2.12.17"
   val scala213 = "2.13.9"
-  val scala3 = "3.1.3"
+  val scala3 = "3.2.0"
```

Comment:
why do we need to bump this? it should be fine as the [compatibility plans are still being ironed](https://www.scala-lang.org/blog/2022/08/17/long-term-compatibility-plans.html) (so it's only starting 3.0 that we should pin), but I would not bump in this PR unless we fully understand the cost/benefits.